thinc/tests/model/test_model.py

```diff
@@ -404,15 +405,17 @@ def get_model_id(id_list, index):
     assert len(list_of_ids) == len(list(set(list_of_ids)))
 
 
+@pytest.mark.skipif(not gpu_is_available(), reason="needs GPU")
```

Comment:
I'd expect

```suggestion
@pytest.mark.skipif(not (has_tensorflow_gpu and has_cupy_gpu)), reason="needs GPU")
```

Or something like that.

We should avoid `gpu_is_available` for specific tests, since it will also return `True` in the future for non-CUDA GPUs (e.g. MPS).